src/server/lib/format.js

```diff
@@ -13,32 +16,39 @@ const EXPENDITURE_CATEGORIES = {
   ec7: '7-Administrative and Other'
 }
 
-function currency (value, optional = false) {
-  if (optional && value == null) {
-    return value
-  }
+/**
+ * Normalize casing of single word values.
+ * This is especially useful for "Yes"/"No" responses.
+ *
+ * @param {string} value
+ * @returns {string}
+ */
+function capitalize (value) {
+  if (value == null) return value
```

Comment:
Probably need to check for the empty string here as well, or the `value[0].toUpperCase()` could throw an exception

Reply:
In JS, `''[0]` just returns undefined.

I'm not sure it's worth being too defensive in this code path.  If we encounter invalid input values, the right place to warn users is the validation stage.  Once we get here, we need to output something.

Reply:
the index won't throw, but calling toUpperCase on the undefined response will.

I'm not too worried about having particualarly strong validations here, but it seems like a weird edge case where a null value is fine but an empty string value crashes is nonintuitive and likely to cause confusion at some point.

Reply:
Oops, great point.  I guess I better fix that!